polaris-react/src/components/ActionList/components/SearchField/SearchField.scss

```diff
@@ -213,24 +164,20 @@ $search-icon-width-se23: calc(#{$icon-size-se23} + var(--p-space-3));
 
 .Backdrop {
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  @include focus-ring($border-width: 1px);
+  @include no-focus-ring;
```

Comment:
Used `outline` and `outline-offset` styles instead of `focus-ring` mixin on line 181-182 to fix focus ring styles.

Before:
<img width="768" alt="actionlist-before-22" src="https://github.com/Shopify/polaris/assets/26749317/2333a751-785c-41a7-ad4a-04c52fa6853b">

After:
<img width="774" alt="actionlist-after-22" src="https://github.com/Shopify/polaris/assets/26749317/9a4e3b48-bfa6-4307-b35c-659f7387e2d1">